src/app/main.component.html

```diff
@@ -0,0 +1,29 @@
+<td-layout #layout>
+  <td-navigation-drawer logo="assets:teradata-dark" sidenavTitle="SANDBOX" name="Firstname Lastname" email="firstname.lastname@company.com">
```

Comment:
Name and email not applicable.

Reply:
They apply to the patterns/nav drawer demo and are standard in all Covalent apps (though I agree, they don't apply since there's no user account associated with the sandbox...)